app/src/main/java/com/example/poste/models/User.java

```diff
@@ -72,7 +72,7 @@ public static User getUser() {
      * Adds a folder to the user's folders.
      * @param newFolder the folder to add to the user's folders.
      */
-    private void addFolder(Folder newFolder) {
+    public void addFolder(Folder newFolder) {
```

Comment:
Should ideally be an API call, as above

Reply:
All i did here was make this public instead of private (for the debugger method).

This is the method you made for building the data modal from the API response, not one that the program will use to make a new folder.
so once a folder is built from the API response, this is simply telling the app to include it in the list of folders. 
at least that's what I figured it was for.
![image](https://github.com/Appyo-Poste/Poste/assets/127574656/dc57f6c0-f14a-4815-ade4-b7334177d569)

Reply:
I see. No worries. I thought you were using this in another fashion, misunderstood.